pkg/playground/manifest.json.in

```diff
@@ -23,6 +23,12 @@
         },
         "preloaded": {
             "label": "Preloaded"
+        },
+        "notifications": {
+            "label": "Notifications"
+        },
+        "notifications-receiver": {
+            "label": "Notifications Receiver"
```

Comment:
I would have expected that this gets preloaded, and that the receiver handles "preloaded" and "fully rendered" differently?

Reply:
Hmm, preloading makes sense for a pages that produce notifications, but it's alright for a consumer to only load when the user switches to it.